source/PlayerInfo.cpp

```diff
@@ -200,7 +200,6 @@ void PlayerInfo::Load(const string &path)
 		else if(child.Token(0) == "mission")
 		{
 			missions.emplace_back(child);
-			cargo.AddMissionCargo(&missions.back());
```

Comment:
This doesn't seem right. When the player lands on a planet, all cargo from all landed ships is transferred to the player's pooled cargohold. When the player departs a planet, first the game is saved, and then during the TakeOff code, it divvies up the pooled cargo into individual ships. That means any mission cargo or passengers that were in-system are now irreparably lost, and you wouldn't be able to complete those missions. They also wouldn't fail, because no ship was destroyed that had some mission cargo, so the player would just be stuck wondering why a mission wouldn't complete, even if every single ship they had was now landed in system.

Reply:
You are correct. I just rewrote most of the changes to correct that bug.

Reply:
I rewrote most of my changes to fix this bug.